src/bigint.cpp

```diff
@@ -0,0 +1,25 @@
+#include "bigint.hpp"
+#include <cmath>
+#include <regex>
+
+namespace libbig
+{
+  largeInt::largeInt() : sign(true), number("0") {}
```

Comment:
ctors are also linked with #16 , but the regex validation for string parameter is missing there, i will merge this after merging #16 

after merging #16 you should modify this file in accordance with #16 inorder to avoid merge conflicts.